icechunk/tests/test_virtual_refs.rs

```diff
@@ -1,910 +1,885 @@
-#[cfg(test)]
```

Comment:
this file didn't really change, I'm only removing the tests module wrapper that was here for some reason.